specification/attestation/resource-manager/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -591,29 +599,166 @@
       "properties": {
         "location": {
           "type": "string",
-          "description": "The supported Azure location where the attestation service instance should be created."
+          "description": "The supported Azure location where the attestation provider should be created."
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "The tags that will be assigned to the attestation service instance."
+          "description": "The tags that will be assigned to the attestation provider."
         },
         "properties": {
           "$ref": "#/definitions/AttestationServiceCreationSpecificParams",
-          "description": "Properties of the attestation service instance"
+          "description": "Properties of the attestation provider"
         }
       }
     },
     "AttestationServiceCreationSpecificParams": {
-      "description": "Client supplied parameters used to create a new attestation service instance.",
+      "description": "Client supplied parameters used to create a new attestation provider.",
       "properties": {
         "policySigningCertificates": {
           "description": "JSON Web Key Set defining a set of X.509 Certificates that will represent the parent certificate for the signing certificate used for policy operations",
           "$ref": "../../../../../common-types/rfcs/rfc7517.json#/definitions/JSONWebKeySet"
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionItem"
```

Comment:
We can just ref to here, here is root entry.  rest of definition is not required if so. right?
 "$ref": "../../../../../common-types/resource-management/v1/privatelinks.json#/definitions/PrivateEndpointConnection"